internal/handlers/organization.go

```diff
@@ -35,3 +41,54 @@ func getOrganization(w http.ResponseWriter, r *http.Request) {
 		RespondJSON(w, organization)
 	}
 }
+
+func updateOrganization(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	auth := auth.Authentication.Require(ctx)
+
+	organization, err := JsonBody[types.Organization](w, r)
+	if err != nil {
+		return
+	} else if organization.Name == "" {
+		http.Error(w, "name is required", http.StatusBadRequest)
+		return
+	}
+
+	if existingOrganization, err := db.GetOrganizationByID(ctx, *auth.CurrentOrgID()); err == nil {
+		if existingOrganization.Slug != nil {
+			if organization.Slug == nil || *organization.Slug == "" {
+				http.Error(w, "Slug can not get unset", http.StatusBadRequest)
+				return
+			}
+
+			if matched, _ := regexp.MatchString("^[a-z]+$", *organization.Slug); !matched {
```

Comment:
The distribution spec has this pattern for the manifest `<name>`: 

`[a-z0-9]+((\.|_|__|-+)[a-z0-9]+)*(\/[a-z0-9]+((\.|_|__|-+)[a-z0-9]+)*)*`

We could use the first part (`[a-z0-9]+((\.|_|__|-+)[a-z0-9]+)*`) for the organization slug.

Also, I think it would make sense to limit the length. 64 characters? 256 characters?